dateparser/utils/__init__.py

```diff
@@ -133,6 +135,24 @@ def apply_timezone_from_settings(date_obj, settings):
     return date_obj
 
 
+def get_last_day_of_month(year, month):
+    return calendar.monthrange(year, month)[1]
+
+
+def set_correct_day_from_settings(date_obj, settings, last_day=None, current_day=None):
```

Comment:
:lipstick: What about `apply_day_of_the_month_from_settings`, to be consistent with `apply_timezone_from_settings`?

Reply:
Hmmm... This was my first approach, but then I didn't like that naming. First of all "apply" seems to mean that you are "adding" something. It maybe has sense for the timezone, as sometimes you add it, but I'm not sure if this has sense for the day as what you are doing is to change it.

Secondly, `apply_day_of_the_month_from_settings` seems to mean that it is getting the day from the settings; but what it is doing is using the rules in the settings to find the correct day.

For both reasons I prefer this naming: `set_correct_day_from_settings`.

What do you think @Gallaecio ? Any better idea?